test/v1/connectors/AmendEmploymentExpensesConnectorSpec.scala

```diff
@@ -50,29 +49,18 @@ class AmendEmploymentExpensesConnectorSpec extends ConnectorSpec {
       appConfig = mockAppConfig
     )
 
```

Comment:
Just to make this file consistent with other endpoint, should we make it `trait Test { _: ConnectorTest =>` instead of a class?